src/Mapping/Driver/AbstractAnnotationDriver.php

```diff
@@ -133,4 +133,20 @@ protected function getRelatedClassName($metadata, $name)
 
         return class_exists($className) ? $className : '';
     }
+
+    /**
+     * Set the setter method for the given field.
+     */
+    protected function setSetterMethod(string $field, ?string $method, array &$config): void
+    {
+        if (empty($method)) {
```

Comment:
```suggestion
        if ('' === $method) {
```

Reply:
`\Gedmo\Mapping\Annotation\Blameable::$setterMethod` is nullable and null by default. I can ensure it's always a string if you want.